server/jwt.go

```diff
@@ -79,31 +79,40 @@ func GenerateTokensAndSetCookies(c echo.Context, user *api.Principal, mode commo
 	return nil
 }
 
+// generateAccessTokenWithoutExpirationTime generates the JWT access token without the expiration time.
+// Used to generate the token for ServiceAccount to provide the token for OpenAPI.
+func generateAccessTokenWithoutExpirationTime(user *api.Principal, mode common.ReleaseMode, secret string) (string, error) {
```

Comment:
The service account should have expiration time and the client should be able to retry on expired token.